src/components/Reports/HealthIndicatorReport/HealthIndicatorGraph/useGraphData.ts

```diff
@@ -0,0 +1,76 @@
+import { DateTime } from 'luxon';
+import { useLocale } from 'src/hooks/useLocale';
+import { monthYearFormat } from 'src/lib/intlFormat';
+import { isNotNullish } from 'src/lib/typeGuards';
+import { useHealthIndicatorGraphQuery } from './HealthIndicatorGraph.generated';
+
+export interface Period {
+  month: string;
+  consistencyHi: number | null | undefined;
+  consistencyHiScaled: number | null | undefined;
+  depthHi: number | null | undefined;
+  depthHiScaled: number | null | undefined;
+  ownershipHi: number | null | undefined;
+  ownershipHiScaled: number | null | undefined;
+  successHi: number | null | undefined;
+  successHiScaled: number | null | undefined;
+}
+
+interface UseGraphDataResult {
+  loading: boolean;
+  average: number | null;
+  periods: Period[] | null;
+}
+
+// Scale a health indicator value by its weight in the overall calculation
+const scale = (
+  value: number | null | undefined,
+  weight = 1,
+): number | null | undefined => {
+  return typeof value === 'number' ? (value * weight) / 7 : value;
+};
+
+export const useGraphData = (accountListId: string): UseGraphDataResult => {
+  const locale = useLocale();
+
+  const { data, loading } = useHealthIndicatorGraphQuery({
+    variables: {
+      accountListId,
+    },
+  });
+
+  const average = data
+    ? Math.round(
+        data.healthIndicatorData
+          .map((month) => month.overallHi)
+          .filter(isNotNullish)
+          .reduce(
+            (total, overallHi, _index, months) =>
+              total + overallHi / months.length,
+            0,
+          ),
+      )
+    : null;
+
+  const periods =
+    data?.healthIndicatorData.map((month) => ({
+      month: monthYearFormat(
+        DateTime.fromISO(month.indicationPeriodBegin),
+        locale,
+      ),
+      consistencyHi: month.consistencyHi,
```

Comment:
This might sound silly, but is there a reason why they are called `consistencyHi` and not just `consistency`? 

I thought `Hi` was sort for `high`. We won;t be using these values elsewhere, so we could just remove `Hi`, unless you think we need to keep it?

Reply:
I was just using the names of the GraphQL fields. But I can remove `Hi` if you want.

```graphql
type HealthIndicatorData {
  calculatedAt: ISO8601DateTime
  consistencyHi: Int
  createdAt: ISO8601DateTime!
  depthHi: Int
  designationAccountNumber: String!
  id: ID!
  indicationPeriodBegin: ISO8601Date!
  machineCalculatedGoal: Int
  overallHi: Int
  ownershipHi: Int
  staffEnteredGoal: Int
  staffUnitId: ID!
  successHi: Int
  updatedAt: ISO8601DateTime!
}
```